src/SpectatorView.Unity/Assets/SpatialAlignment/Scripts/Utilities/SpatialCoordinateLocalizer.cs

```diff
@@ -1,144 +1,144 @@
-
-                    if (showDebugVisuals)
-                    {
-                        if (debugVisual != null)
-                        {
-                            if (debugGameObject == null)
-                            {
-                                debugGameObject = Instantiate(debugVisual);
-                            }
-
-                            debugGameObject.transform.parent = targetRoot.transform;
-                            debugGameObject.transform.position = position;
-                            debugGameObject.transform.rotation = rotation;
-                            debugGameObject.transform.localScale = debugVisualScale * Vector3.one;
-                            debugGameObject.name = $"{targetRoot.name} - SpatialCoordinateLocalizer DebugVisual";
-                            DebugLog($"Created {debugGameObject.name}, World Space Position: {position.ToString("G4")}, World Space Rotation {rotation.ToString("G4")}");
-                        }
-                        else
-                        {
-                            DebugLog($"SpatialCoordinateLocalizer for {targetRoot.name}, Coordinate id: {spatialCoordinate.Id} had showDebugVisuals enabled but no debugVisual specified.");
-                        }
-                    }
-                }
-            }
-        }
-        private ISpatialCoordinate spatialCoordinate = null;
-        private GameObject debugGameObject = null;
-
-        private void Awake()
-        {
-            if (targetRoot is null)
-            {
-                targetRoot = gameObject;
-            }
-        }
-
-        private void Update()
-        {
-            bool isEnabled = (Coordinate?.State ?? LocatedState.Tracking) == LocatedState.Tracking;
-
-            if (isEnabled)
-            {
-                targetRoot.transform.position = Coordinate?.CoordinateToWorldSpace(CoordinateRelativePosition) ?? targetRoot.transform.position;
-                targetRoot.transform.rotation = Coordinate?.CoordinateToWorldSpace(CoordinateRelativeRotation) ?? targetRoot.transform.rotation;
-            }
-
-            if (autoToggleActive)
-            {
-                targetRoot.SetActive(isEnabled);
-            }
-        }
-
-        private void DebugLog(string message)
-        {
-            if (debugLogging)
-            {
-                Debug.Log($"SpatialCoordinateLocalizer: {message}");
-            }
-        }
-    }
-}
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Experimental.SpatialAlignment.Common
+{
+    /// <summary>
+    /// Very simple consumer of <see cref="ISpatialCoordinate"/> to demonstrate usage.
+    /// </summary>
+    public class SpatialCoordinateLocalizer : MonoBehaviour
```

Comment:
>SpatialCoordinateLocalizer [](start = 17, length = 26)

nit: spectator view doesn't currently use this file. If we want ot keep it, it should probably go the in the sharing.spatialalignment directories.